lib/mconf/ldap.rb

```diff
@@ -10,8 +10,7 @@ class LDAP
     # the root key used to store all the information in the session
     ENV_KEY = :ldap_data
 
-    # `session` is the session object where the user information will be stored
-    def initialize(session)
+    def initialize session={}
```

Comment:
Note for future PRs: https://github.com/bbatsov/ruby-style-guide#method-parens :)